README.md

```diff
@@ -1 +1,305 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+    <li><a href="https://react.dev/">React</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Tests / API</summary>
+  <ul>
+    <li><a href="https://rspec.info/">Rspec</a></li>
+    <li><a href="https://github.com/rswag/rswag/">Rswag</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+The main features of this app include:
+
+- **Load, create, and delete doctors**
+- **Load, create and delete appointments / reservations**
+- **Signup and Signin feature**
+- **Admin and User roles**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- Fronend Version -->
+
+### Frontend Version <a name="frontend-version"></a>
+
+- [Frontend version](https://github.com/katarighe/book-appointment-front-end)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Kanban Board -->
+
+### Kanban Board <a name="kanban-board"></a>
+
+- [Original Kanban Board](https://github.com/users/katarighe/projects/4/views/1)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to install the following items:
+
+- Ruby
+- Ruby on Rails
+- PostgreSQL
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone git@github.com:katarighe/book-appointment-back-end.git
+cd book-appointment-back-end
+```
+
+### Install
+
+Install this project by using:
+
+```
+bundle install
+```
+
+Setup database with:
+
+```
+rails db:create
+rails db:migrate
+rails db:seed
+```
+
+### Usage
+
+To run the project, execute the following command to start the server:
+
+```
+rails server -p 3001
+```
+
+Visit http://localhost:3001/ in your browser.
+
+### Run tests
+
+To run tests, run the following commands:
+
+Install npm with:
+
+```
+npm install
+```
+
+Install rspec with:
+
+```
+bundle install
+```
+
+and
+
+```
+rails generate rspec:install
+```
+
+run the test with:
+
+```
+rspec spec
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

Reply:
✅ Done, thanks for the help.